src/scripts/record-books/record-books.component.js

```diff
@@ -107,8 +108,10 @@ function RecordBooksController($scope, dimStoreService, dimDefinitions, dimSetti
       const objectiveDef = defs.Objective.get(objective.objectiveHash);
 
       let display = undefined;
-      if (record.recordValueUIStyle === '_investment_record_value_ui_style_time_in_milliseconds') {
-        display = objective.displayValue;
+      if (recordDef.recordValueUIStyle === '_investment_record_value_ui_style_time_in_milliseconds') {
+        display = objective.isComplete
+          ? objective.displayValue
+          : moment(moment.duration(objectiveDef.completionValue)._data).format("m:ss.SSS");
```

Comment:
bit of a hack to access the private ._data property, but it seems this is how people do this. Suppose I could do the math myself instead of using moment.... :)